kythe/go/platform/tools/indexer_wrapper/local.go

```diff
@@ -0,0 +1,99 @@
+// This package provides an implementation of the CompilationAnalyzer interface
```

Comment:
This isn't a separate package from the binary, but it probably should be.  It could go in kythe/go/platform/analysis/local/local.go.